src/BioFSharp.BioTools/MoFF.fs

```diff
@@ -0,0 +1,345 @@
+            | RawFilesFolder (path)               -> ["--raw_repo" ; (MountInfo.containerPathOf m path)]
+            | Ms2PeptidesInputFile (pathList)     -> List.append ["--tsv_list"] (pathList |> List.map (fun path -> MountInfo.containerPathOf m path))
+            | RawFile (pathList)                  -> List.append ["--raw_list"] (pathList |> List.map (fun path -> MountInfo.containerPathOf m path))
+
+        static member make = function
+            | Ms2PeptidesInputFileFolder (path)   -> ["--loc_in" ; path]
+            | RawFilesFolder (path)               -> ["--raw_repo" ; path]
+            | Ms2PeptidesInputFile (pathList)     -> List.append ["--tsv_list"] pathList 
+            | RawFile (pathList)                  -> List.append ["--raw_list"] pathList 
+
+    type MatchBetweenRunsOptions =
+        ///--sample                      reg exp to filter the input file names (only with --loc_in input option-
+        | FilterInputFiles of string
+        ///--ext                         file extention of the input file. Default .txt)
+        | FileExtensionsInputFile of string
+        ///--log_label                   filename for the mbr log file. Default moFF_mbr
+        | FileNameMBRLogFile of string
+        ///--w_filt                      width value for outlier filtering. Default 3
+        | WidthForOutlierFiltering of int
+        ///--out_flag                    if set, outliers for rt time allignment are filtered. Default value: True
+        | OutlierFilter of bool
+        ///--w_comb                      if set, RT model combination is weighted using traing model errors: Default value: False
+        | WeightedRTModelCombination of bool
+
+        static member makeWith (m: MountInfo) = function
+            | FilterInputFiles (name)                 -> ["--sample" ; string name]
+            | FileExtensionsInputFile (fileExtension) -> ["--ext" ; string fileExtension]
+            | FileNameMBRLogFile (name)               -> ["--log_label" ; string name]
+            | WidthForOutlierFiltering (width)        -> ["--w_filt" ; string width]
+            | OutlierFilter (boolean)                 -> ["--out_flag" ; string boolean]
+            | WeightedRTModelCombination (boolean)    -> ["--w_comb" ; string boolean]
+
+        static member make = function
+            | FilterInputFiles (name)                 -> ["--sample" ; string name]
+            | FileExtensionsInputFile (fileExtension) -> ["--ext" ; string fileExtension]
+            | FileNameMBRLogFile (name)               -> ["--log_label" ; string name]
+            | WidthForOutlierFiltering (width)        -> ["--w_filt" ; string width]
+            | OutlierFilter (boolean)                 -> ["--out_flag" ; string boolean]
+            | WeightedRTModelCombination (boolean)    -> ["--w_comb" ; string boolean]
+
+    ///mzML must be specified using --tsv_list | --raw_list (Ms2PeptidesInputFile & RawFile)
```

Comment:
Additionally to this text, add suggestions from above

Reply:
done